bin/list-all

```diff
@@ -3,9 +3,9 @@
 releases_url="https://api.github.com/repos/mvdan/sh/releases"
 
 if [ -n "$GITHUB_API_TOKEN" ]; then
-	cmd="curl -s -H 'Authorization: token $GITHUB_API_TOKEN' $releases_url"
+	cmd="curl --retry 10 --retry-delay 2-s -H 'Authorization: token $GITHUB_API_TOKEN' $releases_url"
```

Comment:
missing a space here `2-s`

Reply:
@luizm apologies, I missed this notification. Just pushed the fix.

Reply:
thank you ❤️ to be honest I forgot about this PR.. my bad